docs: german translations (#548)
Tresjs/tres: docs/de/examples/text-3d.md

Allerdings ist es kein Teil des Kerns von Three.js. Daher müsstest du es aus dem Modul `three/addons/geometries/TextGeometry` importieren.

Das schafft ein Problem, weil **TresJS** automatisch einen Katalog aus dem Kern von Three erstellt, damit du sie als Komponenten nutzen kannst.

Glücklicherweise bietet **TresJS** eine Möglichkeit, den Komponentenkatalog zu erweitern. Du kannst dies tun, indem du die `extend`-Methode der Hauptbibliothek verwendest.

Für weitere Informationen darüber, wie du deinen TresJS-Katalog erweitern kannst, siehe den Abschnitt [Erweitern](/de/advanced/extending.md).

## Verwendung von TextGeometry

Um `TextGeometry` zu verwenden, musst du es aus dem Modul `three/addons/geometries/TextGeometry` importieren.

```js
import { TextGeometry } from 'three/addons/geometries/TextGeometry'
```

Danach musst du den Komponentenkatalog mit der `extend`-Methode erweitern.

```js
import { extend } from '@tresjs/core'
import { TextGeometry } from 'three/addons/geometries/TextGeometry'

extend({ TextGeometry })
```

> [TextGeometry](https://threejs.org/docs/index.html?q=text#examples/en/geometries/TextGeometry) benötigt nur ein Argument, die Schriftart. Unten siehst du ein Beispiel.

Reviewer: ```suggestion
[TextGeometry](https://threejs.org/docs/index.html?q=text#examples/en/geometries/TextGeometry) benötigt nur die Schriftart als Argument. Unten siehst du ein Beispiel.
```

const fontOptions = {
  size: 0.5,
  height: 0.2,
  curveSegments: 5,
  bevelEnabled: true,
  bevelThickness: 0.05,
  bevelSize: 0.02,
  bevelOffset: 0,
  bevelSegments: 4,
}

const loader = new FontLoader()

const font = await new Promise((resolve, reject) => {
  try {
    loader.load(fontPath, (font) => {
      resolve(font)
    })
  }
  catch (error) {
    reject(console.error('cientos', error))
  }
})

const matcapTexture = await useTexture(['https://raw.githubusercontent.com/Tresjs/assets/main/textures/matcaps/7.png'])
</script>

<template>
  <TresCanvas
    shadows
    alpha
  >
    <TresMesh>
      <TresTextGeometry
        :args="['TresJS', { font, ...fontOptions }]"
        center
      />
      <TresMeshNormalMaterial :matcap="matcapTexture" />
    </TresMesh>
  </TresCanvas>
</template>
```

Das scheint viel Arbeit zu sein, aber es gibt eine einfachere Möglichkeit.

## TextGeometry von `cientos`

Das Paket `cientos` bietet eine Komponente namens `<Text3D />`, die ein Wrapper für `TextGeometry` aus dem Modul [`three-stdlib`](https://github.com/pmndrs/three-stdlib) ist.

Das Beste daran? Du musst den Katalog nicht erweitern, gib einfach das Schriftargument an.

> Es funktioniert einfach. 💯 (wenn kein Text bereitgestellt wird, wird der Text TresJS sein)

Reviewer: ```suggestion
Es funktioniert einfach. 💯 (wenn kein Text angegeben wird, wird der Text "TresJS" sein)
```

```vue
<template>
  <TresCanvas
    shadows
    alpha
  >
    <Text3D :font="fontPath" />
  </TresCanvas>
</template>
```

Wir können die Optionen als Props übergeben

```html
<Text3D :font="fontPath" :text="my 3d text" :size="0.8" />
```

falls die Optionen nicht bereitgestellt werden, sind die Standardwerte:

```js
const fontOptions = {
  size: 0.5,
  height: 0.2,
  curveSegments: 5,
  bevelEnabled: true,
  bevelThickness: 0.05,
  bevelSize: 0.02,
  bevelOffset: 0,
  bevelSegments: 4,
}
```

> Standardmäßig beginnt der Text in Three.js an der Ausgangsposition des Meshs, also wenn es [0,0,0] ist, wird der Text dort beginnen, aber wir können ihn einfach zentrieren, indem wir das Flag "center" übergeben.

Reviewer: ```suggestion
Der Text in Three.js beginnt standardmäßig an der Ausgangsposition des Meshs, also bei [0,0,0]. Um den Text zu zentrieren, können wir einfach das Flag "center" übergeben.
```
